src/meta/processors/partsMan/DropSpaceProcessor.cpp

```diff
@@ -96,10 +97,29 @@ void DropSpaceProcessor::process(const cpp2::DropSpaceReq& req) {
         lstIter->next();
     }
 
-    // Delete statis data if it exists
+    // 5. Delete related statis data
     auto statiskey = MetaServiceUtils::statisKey(spaceId);
     deleteKeys.emplace_back(statiskey);
 
+    // 6. Delete related leader data
```

Comment:
IMO, we don't need this step, even if we delete here, subsequent HB will write the leader key as well. Just delete them in `show hosts` is enough.